setup.py

```diff
@@ -1,42 +1,151 @@
+import distutils
 import glob
 import os
+import shlex
 import setuptools
 import setuptools.command.build_ext
+import setuptools.command.develop
+import setuptools.command.install
 import setuptools.dist
 import sys
 import setuptools.command.install
 import cellprofiler.utilities.version
 
+# Recipe needed to get real distutils if virtualenv.
```

Comment:
The error messages show up when you run the distribution's "cellprofiler.exe" after running `python setup.py py2exe` within a virtual environment on the build machine.

h5py.version imports distutils.version and the virtualenv hack causes the "cannot import name dist", so it's not our use of distutils that requires the code. The second error happens because py2exe packages the virtualenv's site.py which won't work without a hand-crafted orig-prefix.txt file.